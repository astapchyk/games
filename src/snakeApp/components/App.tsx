import React, { useEffect, ReactElement } from 'react';

import Area from './Area/index';

import styles from './styles.css';

function App(): ReactElement {
  useEffect(() => {
    document.title = 'Snake game';
    return;
  }, []);

  return (
    <div className={styles.container}>
      <span className={styles.title}>
        <h1>Snake App</h1>
      </span>
      <div className={styles.content}>
        <Area />
      </div>
      <span className={styles.score} />
    </div>
  );
}

export default App;
