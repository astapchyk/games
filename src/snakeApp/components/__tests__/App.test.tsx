import React from 'react';
import { render } from '@testing-library/react';

import App from '../App';

describe('<App />', () => {
  test('renders the component', async () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
