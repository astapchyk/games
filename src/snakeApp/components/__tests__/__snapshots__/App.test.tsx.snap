// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the component 1`] = `
<div>
  <div
    class="container"
  >
    <span
      class="title"
    >
      <h1>
        Snake App
      </h1>
    </span>
    <div
      class="content"
    >
      <div
        class="cell snake"
        style="left: 75vh; top: 75vh;"
      />
    </div>
    <span
      class="score"
    />
  </div>
</div>
`;
